src/backend/src/routes/users.routes.ts

```diff
@@ -54,11 +54,8 @@ userRouter.post(
   validateInputs,
   UsersController.getManyUserTasks
 );
-userRouter.get('/:userId/notifications', UsersController.getUserUnreadNotifications);
-userRouter.post(
-  '/:userId/notifications/remove',
-  nonEmptyString(body('notificationId')),
-  UsersController.removeUserNotification
-);
+userRouter.get('/notifications/current-user', UsersController.getUserUnreadNotifications);
```

Comment:
I think it would make more sense to create new routers rather than going this deep
something like `notificationRouter.get('current-user')`